_episodes/05-function-style.md

```diff
@@ -72,34 +89,48 @@ canvas(with_attribution=True)
 ~~~
 {: .output}
 
-If we try the same thing on our `mean` function, we don't get a helpful error message.
+If we try the same thing on our `open_pdb` function, we don't get a helpful error message.
 
-We will want to write a docstring for our new mean function. This way, it will be clear to new developers who use our code what the function does, and be accessible to any users using the function interactively. Returning, to the `molssi_math.py` module file, edit your `mean` function to look like the following:
+We will want to write a docstring for our new mean function. This way, it will be clear to new developers who use our code what the function does, and be accessible to any users using the function interactively. Returning, to the `pdb.py` module file, edit your `open_pdb` function to look like the following:
 
 ~~~
-def mean(num_list):
-    """
-    Calculate the mean/average of a list of numbers.
+def open_pdb(file_location):
+    """Open and read coordinates and atom symbols from a pdb file.
+
+    The pdb file must specify the atom elements in the last column, and follow
+    the conventions outlined in the PDB format specification.
 
     Parameters
     ----------
-    num_list : list
-        The list to take the average of
+    file_location : str
+        The location of the xyz file to read in.
 
     Returns
     -------
-    mean_list: float
-        The mean of the list
+    coords : np.ndarray
+        The coordinates of the xyz file.
+    symbols : list
+        The atomic symbols of the xyz file.
 
-    Examples
-    --------
-    >>> mean([1, 2, 3, 4, 5])
-    3.0
     """
 
-    mean_list = sum(num_list)/len(num_list)
+    with open(file_location) as f:
+        data = f.readlines()
+
+    coordinates = []
+    symbols = []
+
+    for line in data:
+        if 'ATOM' in line[0:6] or 'HETATM' in line[0:6]:
+            symbols.append(line[76:79].strip())
+
+            coords = [float(x) for x in line[30:55].split()]
+            coordinates.append(coords)
+
+    coords = np.array(coordinates)
+    symbols = np.array(symbols)
```

Comment:
Don't correct the way the function is written yet. This should be redone when talking about PEP8.